aten/src/ATen/templates/RegisterFunctionalization.cpp

```diff
@@ -94,6 +94,11 @@ inline c10::List<::std::optional<Tensor>> to_meta(const c10::List<::std::optiona
   return outputs;
 }
 
+static bool disable_meta_reference() {
+  static auto env = std::getenv("TORCH_DISABLE_FUNCTIONALIZATION_META_REFERENCE");
```

Comment:
There will be one static variable for each `RegisterFunctionalization_X.cpp` file. While this is not ideal, I think this should reduce function call overhead.